package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "building restaurant page",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Byabasaija/Restaurant-page.git"
+  },
+  "author": "pascal byabasaija",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Byabasaija/Restaurant-page/issues"
+  },
+  "homepage": "https://github.com/Byabasaija/Restaurant-page#readme",
+  "devDependencies": {
+    "babel-eslint": "10.1.x",
+    "eslint": "6.8.x",
+    "eslint-config-airbnb-base": "14.1.x",
+    "eslint-plugin-import": "2.20.x",
+    "stylelint": "13.3.x",
+    "stylelint-config-standard": "20.0.x",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "3.17.x",
+    "webpack": "^5.26.2",
+    "webpack-cli": "^4.5.0"
+  }
+}
```

Comment:
- [ ] You are missing the scripts keys here with the commands to run your application with Webpack, here's a basic example:

```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```